library/src/main/java/ch/decent/sdk/model/ObjectType.kt

```diff
@@ -38,7 +38,8 @@ enum class ObjectType {
   SUBSCRIPTION_OBJECT, //15
   SEEDING_STATISTICS_OBJECT,
   TRANSACTION_DETAIL_OBJECT,
-  MESSAGING_OBJECT;
+  MESSAGING_OBJECT,
+  UNKNOWN_OBJECT;
 
   val space: Byte
     get() = if (ordinal < 10) 1 else 2
```

Comment:
Magic number '10' shouldn't be used, replace with ordinal of GLOBAL_PROPERTY_OBJECT or don't use ordinal